src/main/java/org/prebid/server/floors/PriceFloorFetcher.java

```diff
@@ -78,44 +80,48 @@ public PriceFloorFetcher(ApplicationSettings applicationSettings,
                 .asMap();
     }
 
-    public PriceFloorRules fetch(Account account) {
+    public FetchResult fetch(Account account) {
+        if (account == null) {
```

Comment:
Don't think we should cover this case.
Assumed, account is always present and if it absent - throwing NPE is fine since it will be server side issue to observe in logs.